src/SSW.SophieBot/SSWSophieBot/dialogs/GetExpertsByLocationDialog/GetExpertsByLocationDialog.dialog

```diff
@@ -42,7 +38,10 @@
           "statusCodeProperty": "turn.statusCode",
           "reasonPhraseProperty": "turn.reasonPhrase",
           "employeesProperty": "turn.employees",
-          "skill": {}
+          "skill": {
+            "technology": "=dialog.skillEntity",
+            "experienceLevel": "Advanced"
```

Comment:
Data is not correct. In `GetProfileAction`, people with the `Intermediate` level will be filtered if you pass the skill above. You can try `who works in Sydney?`, it will show **6** people with Azure skills. Actually, it should be **9**.
![image](https://user-images.githubusercontent.com/37203901/134795823-eeae596a-7d09-4823-bbe9-ac931085887f.png)

![image](https://user-images.githubusercontent.com/37203901/134795643-b263e4f7-d60b-462a-88ea-a592a4abead2.png)

![image](https://user-images.githubusercontent.com/37203901/134795659-b58c78a9-2ab2-44ee-9b3d-d4e028b4f38e.png)